File: fastmq-storage/src/main/java/com/song/fastmq/storage/impl/InvalidLedgerException.java
package com.song.fastmq.storage.impl;

/**
 * @author song
 */
public class InvalidLedgerException extends Exception {

    public InvalidLedgerException() {
    }

    public InvalidLedgerException(String message) {
        super(message);
    }

    public InvalidLedgerException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: fastmq-storage/src/main/java/com/song/fastmq/storage/storage/support/InvalidLedgerException.kt
package com.song.fastmq.storage.storage.support

/**
 * @author song
 */
class InvalidLedgerException : Exception {

    constructor() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}
}
